deploy/src/Microsoft.Azure.IIoT.Deployment/Deployment/ApplicationsManager.cs

```diff
@@ -418,7 +418,6 @@ await _msGraphServiceClient
                 $"{applicationUrl}registry/swagger/oauth2-redirect.html",
                 $"{applicationUrl}twin/swagger/oauth2-redirect.html",
                 $"{applicationUrl}history/swagger/oauth2-redirect.html",
-                $"{applicationUrl}vault/swagger/oauth2-redirect.html",
```

Comment:
@marcschier, `Microsoft.Azure.IIoT.Deployment` does not deploy the latest version of the platform from `master`. Instead it deploys `2.7.199` version. So all your changes in `Microsoft.Azure.IIoT.Deployment` project will effectively break deployment of `2.7.199`. It also uses `0.3.1` version of `azure-industrial-iot` Helm chart from our public Helm repository. So we will be able to change deployment code here only after we have published next version of our Helm chart without those three experimental microservice. So for now, I would have to ask you to revert your changes in `Microsoft.Azure.IIoT.Deployment` project and in its documentation in `docs/deploy/howto-deploy-aks.md`. 

It would be great if you can keep only those changes in a separate branch so that we can merge it at a later date when we are ready to have a new release. I can take care of maintaining that branch and rebasing it from time to time. If you want, I can do that myself, but please let me know.

Please note, that what I said above does not apply for the Helm chart, so you do not need to revert anything there.

Reply:
Super - will do and thank for the feedback!